nilearn/plotting/cm.py

```diff
@@ -85,6 +85,30 @@ def _concat_cmap(cmap1, cmap2):
 
     return cdict
 
+def threshold_cmap(cmap, norm, threshold=None):
```

Comment:
I think that this should be a private function (ie start with a "_")

Reply:
done.